lib/app.rb

```diff
@@ -0,0 +1,28 @@
+require 'sinatra/base'
+# require 'byebug'
+# require 'pry'
```

Comment:
Okay, so the reason I comment on that is mainly because these lines, and many others are line-for-line copy of your main controller in the bookmark manager app https://github.com/chrisco/bookmark_manager/blob/master/lib/app.rb. So, my point is that you should not copy-and-paste code but rather write it yourself. It is better for learning and gives you a better opportunity to reflect over the functionality it provides. It is not